### PR TITLE
DOCS-3933: Renaming credit cards

### PR DESCRIPTION
**Pull request by Rendani-MultiSafepay**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

File: content/checkout/payment-components.md [hunk 1]
```diff
@@ -42,9 +42,9 @@ You can embed a single or multiple payment methods in your checkout.
 
 </details>
 
-# Credit card features
+# Card payments features
```
Comment (inesmayerMSP) [inline content/checkout/payment-components.md:45]: payments or payment?

File: content/getting-started/glossary.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ A customer who uses a card issued by a bank to make cashless payments to a merch
 
 ---
 #### card scheme
-A financial institution that issues credit cards, provides infrastructure, and processes payments for a fee, e.g. Visa, Mastercard, American Express.
+A financial institution that issues  credit or debit cards, provides infrastructure, and processes payments for a fee, e.g. Visa, Mastercard, American Express.
```
Comment (inesmayerMSP) [inline content/getting-started/glossary.md:42]: space

File: content/integrations/magento-1.md [hunk 3]
```diff
@@ -987,7 +987,8 @@ All expired orders retain **Waiting** status until you cancel them:
 1. Sign in to your Magento 1 backend.
 2. Go to **Stores** > **Configuration** > **MultiSafepay** > **MultiSafepay settings**.
 
-**Credit cards**
+**Card payments**
```
Comment (inesmayerMSP) [inline content/integrations/magento-1.md:990]: coherence

File: content/integrations/prestashop-1-6.md [hunk 4]
```diff
@@ -167,7 +167,7 @@ Payment links generated manually in your MultiSafepay dashboard don't automatica
 
 1. Sign in to your PrestaShop 1.6 backend. 
 2. Go to **Modules and services** > **Modules and services** 
-3. Select either the bundled credit cards <<glossary:gateway>>, **or** select specific credit cards.
+3. Select either the bundled card payments <<glossary:gateway>>, **or** select specific card payments.
```
Comment (inesmayerMSP) [inline content/integrations/prestashop-1-6.md:170]: specific card

File: content/integrations/woocommerce.md [hunk 5]
```diff
@@ -274,7 +274,7 @@ You need to [activate recurring payments](/docs/recurring-payments/) and then en
 
 1. Sign in to your Wordpress backend.
 2. Go to **WooCommerce** > **MultiSafepay settings** > **Payment methods** 
-3. Select relevant credit cards, and then click **Manage**.
+3. Select relevant card payment, and then click **Manage**.
```
Comment (inesmayerMSP) [inline content/integrations/woocommerce.md:277]: cards

**Review by inesmayerMSP — APPROVED** [review]

File: content/integrations/prestashop-1-6.md (hunk 4 above)
Comment (Rendani-MultiSafepay) [reply content/integrations/prestashop-1-6.md:170]: This is related to the gateway_ID in which the payment method is now changed to card payments from credit cards.

**Pull request merged**